18-additional-resources.Rmd

```diff
@@ -94,12 +96,24 @@ Bryan, J. & Hestor, J. *Happy git and github for the useR*. Retrieved from [http
 
 *Introduction to dplyr*. Retreived from [https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html](https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html)
 
+> `dplyr` is a R package that helps to manipulate and understand data. It provides simple “verbs”, functions that correspond to the most common data manipulation tasks, to help you translate your thoughts into code.Anyone with no programming experience can easily understand the verbs and use them for data analytics with ease.
+
 *A short introduction to the caret package*. Retrieved from [https://cran.r-project.org/web/packages/caret/vignettes/caret.html]([https://cran.r-project.org/web/packages/caret/vignettes/caret.html)
 
+> The `caret` package is an abbreviated form of `Classification And REgression Trees`. It contains functions to streamline the model training process for complex regression and classification problems. The idea behind this package is to streamline the process for creating predictive models. The package contains tools for:
```

Comment:
also needs to be "fresh" language

Reply:
thank you for the feedback. I have rewritten the paragraph.